Backend/Controllers/EnemyController.cs

```diff
@@ -1,58 +1,59 @@
 ﻿using Backend.Services;
-using Domain.Entities;
-using Microsoft.AspNetCore.Cors.Infrastructure;
+using Domain.DTOs;
 
 namespace Backend.Controllers
 {
     public static class EnemyController
     {
         public static void MapEnemyEndpoint(this WebApplication app)
         {
-            //Get all enemies
+            // Get all enemies
             app.MapGet("/enemies", (IEnemyService enemyService) =>
             {
-                return enemyService.GetAllEnemies();
-            }).RequireAuthorization(policy => policy.RequireRole("Admin"));
+                var enemyDTOs = enemyService.GetAllEnemies();
+                return enemyDTOs;
+            });//.RequireAuthorization(policy => policy.RequireRole("Admin"));
 
-            //Get enemy by id
-            app.MapGet("/enemies/{id}", (IEnemyService enemyService, int id) =>
+            // Get enemy by id
+            app.MapGet("/enemy/{id}", (IEnemyService enemyService, int id) =>
```

Comment:
Vi skal være consistent i naming af endpoints, skal være enemies

Reply:
også selvom det kun er en enemy man henter ud?

Reply:
for så er den eneste måde man hurtigt kan se forskel på api'erne at der er {id} efter den

Reply:
Yes, i REST skal alle endpoints være under samme ressource, så det går ikke at der er forskellig naming af enemy. Typisk hvis du rammer en ...com/api/XXX/ med en get, regner du med at få en collection ud af ressourcen

Reply:
Men rigtigt godt spørgsmål